indykite/resource_application_space.go

```diff
@@ -57,11 +59,13 @@ func resAppSpaceCreateContext(ctx context.Context, data *schema.ResourceData, me
 
 	name := data.Get(nameKey).(string)
 	resp, err := clientCtx.GetClient().CreateApplicationSpace(ctx, &config.CreateApplicationSpaceRequest{
-		CustomerId:  data.Get(customerIDKey).(string),
-		Name:        name,
-		DisplayName: optionalString(data, displayNameKey),
-		Description: optionalString(data, descriptionKey),
-		Region:      data.Get(regionKey).(string),
+		CustomerId:    data.Get(customerIDKey).(string),
+		Name:          name,
+		DisplayName:   optionalString(data, displayNameKey),
+		Description:   optionalString(data, descriptionKey),
+		Region:        data.Get(regionKey).(string),
+		IkgSize:       data.Get(ikgSizeKey).(string),
+		ReplicaRegion: data.Get(replicaRegionKey).(string),
```

Comment:
Will this cast to `string` work, if size or replica region is not specified?

Reply:
But I can add the 2 series of tests in the test file